src/coffee/services/subscriptions.coffee

```diff
@@ -0,0 +1,12 @@
+BaseService = require './base'
+
+# IMPORTANT:
+# THIS FEATURE IS FOR INTERNAL USE ONLY.
+# MIGHT BE REMOVED IN THE FUTURE.
+# PLEASE DON'T USE THIS ENDPOINT.
```

Comment:
It clearly states "for internal use ONLY" so why it does not make sense?
What is the difference between this and BETA features (they can have breaking changes also but are supported by SDKs)? We have implemented already first option and i do not like it as it is definitely more work than this tiny PR and does not feel right as we are using this feature in production already.

We did not provide tests on purpose - so that nobody is tempted to try it out on your own already ;)